rbe.bazelrc

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This was initially copied from https://github.com/bazelbuild/bazel-toolchains/blob/master/bazelrc/bazel-1.0.0.bazelrc
+
+# This .bazelrc file contains all of the flags required for the provided
+# toolchain with Remote Build Execution.
+# Note your WORKSPACE must contain an rbe_autoconfig target with
+# name="rbe_default" to use these flags as-is.
+
+# Depending on how many machines are in the remote execution instance, setting
+# this higher can make builds faster by allowing more jobs to run in parallel.
+# Setting it too high can result in jobs that timeout, however, while waiting
+# for a remote machine to execute them.
+build:remote --jobs=50
+
+# Set several flags related to specifying the platform, toolchain and java
+# properties.
+# These flags should only be used as is for the rbe-ubuntu16-04 container
```

Comment:
How about specializing these the way TensorFlow does, with build:rbe, build:rbe_linux, build:rbe_win, etc.?

Reply:
This is just copied from the RBE setup doc instructions, so I thought it was good to leave it for now. I plan to split out the platform-specific stuff and customize the config as we get a more advanced config.

Do you also think we should use `rbe` as the config name instead of `remote`? Again I thought being consistent with the docs would be good, especially since `--config=remote` looks a lot more obvious when you're reading the command than an obscure acronym.

Reply:
Okay, starting simple and not blindly copying TensorFlow SGTM

`rbe` is shorter than `remote` and still gets the point across, if you know the acronym. `remote` also makes me think that the build is for a remote target machine (like `--config=android` means you're building for an Android device)

Reply:
*if you know the acronym* :-p

`--config` is general purpose, so I think you probably shouldn't always read it as specifying a target (e.g. `--config=asan`). Shorter is nice though. Changed.